src/context/apiClient.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import * as rest from "../rest/interface";
+import { ClientConfig, IsValidConfig } from "../models/client_config";
+
+const { createContext, useState } = React;
+
+const configPath = "/config.json";
+
+interface ApiClientContextType {
+  apiClient: rest.Evergreen;
+  actions: {
+    tryLoadConfig: () => void;
+    updateConfig: (configObj: ClientConfig) => void;
+  };
+}
+
+const ApiClientContext = createContext<ApiClientContextType | null>(null);
+
+const ApiClientProvider: React.FC = ({ children }) => {
+  const [apiClient, setApiClient] = useState(
+    rest.EvergreenClient("", "", "", "")
```

Comment:
You need to pass in the values from the config here

Reply:
the value passed to `useState` represents the default state, which is usually null or in this case can be `rest.EvergreenClient` called with empty strings